### DISCUSSION TITLE
Contributors migrated to dev-container

### DISCUSSION THREAD

**Discussion by yecol** [alibaba/GraphScope#2631] [dogfooding]
- [x] @siyuan0322 
- [x] @sighingnow 
- [x] @yecol 
- [x] @lixueclaire 
- [ ] @LiSu 
- [x] @longbinlai 
- [ ] @luoxiaojian 
- [x] @shirly121 
- [x] @BingqingLyu 
- [x] @wenyuanyu 
- [x] @lidongze0629 
- [x] @acezen 
- [x] @dashanji 
- [x] @doudoubobo 
- [x] @septicmk 
- [x] @andydiwenzhu 
- [x] @ds-ssj 
- [ ] @vegetableysm 
- [x] @zhanglei1949 
- [ ] TBC...

**Comment by yecol** [comment]
<img width="821" alt="image" src="https://user-images.githubusercontent.com/426463/234465348-5ad0fe12-1d9e-49c0-9d6e-af68fef7b719.png">

- `quite` maybe not very user friendly? quite for ~10min without any output.

**Comment by yecol** [comment]
No docker installed in the `graphscope-dev`?  it was unable to build images.

```
./gs make-images 
```

**Reply by siyuan0322** [reply]
dev is a container, deploying DinD (Docker in Docker) is not investigated.
